### ISSUE TITLE
Decorator lowering with type arguments produces bad syntax

### ISSUE THREAD

**Issue by alexeagle** [angular/tsickle#138]
This constructor
https://github.com/angular/angular/blob/package-splitup-squashed-rebased/modules/@angular/common/src/directives/ng_for.ts#L88

turns into this lowered annotation:
```
NgFor.ctorParameters = [
    { type: ViewContainerRef, },
    { type: TemplateRef < NgForRow > , },
    { type: IterableDiffers, },
    { type: ChangeDetectorRef, },
];
```

**Comment by mprobst** [comment]
How should a parameterized type be emitted? Is there some sort of Type Token?

**Comment by tbosch** [comment]
As the lowering is only for runtime DI, we don't need the type parameters at all!
I.e. just produce
```
NgFor.ctorParameters = [
    { type: ViewContainerRef, },
    { type: TemplateRef, },
    { type: IterableDiffers, },
    { type: ChangeDetectorRef, },
];
```

**Comment by mprobst** [comment]
https://reviews.angular.io/D101

**Comment by mprobst** [comment]
Fixed in a4804f1c34fe0f2102fb48207d5060d6dabbed77.
